packages/core/database/src/schema/index.ts

```diff
@@ -28,16 +29,14 @@ interface State {
   schema?: Schema;
 }
 
-/**
- * @type {import('.').default}
- */
-export const createSchemaProvider = (db: Database): SchemaProvider => {
+export const createSchemaProvider = (db: Database, options: MetadataOptions): SchemaProvider => {
   const state: State = {};
 
   return {
     get schema() {
       if (!state.schema) {
-        state.schema = metadataToSchema(db.metadata);
+        debug('Creating schema from metadata');
```

Comment:
Could we add a bit more detail about which schema we're creating? Maybe adding the uid somewhere in the log?

Reply:
It's not a specific schema, it's the entire database schema that is all done at once, but I'll clarify that